tests/countries/test_togo.py

```diff
@@ -0,0 +1,260 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.constants import WORKDAY
+from holidays.countries.togo import Togo, TG, TGO
+from tests.common import CommonCountryTests, WorkingDayTests
+
+
+class TestTogo(CommonCountryTests, WorkingDayTests, TestCase):
```

Comment:
```suggestion
from tests.common import CommonCountryTests


class TestTogo(CommonCountryTests, TestCase):
```
`WorkingDayTests` class is used when country has "substituted holidays" (transfer of non-working days from weekend to regular days).